src/shared/components/post/post-listing.tsx

```diff
@@ -1373,7 +1373,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
         <div className="d-block d-sm-none">
           <article className="row post-container">
             <div className="col-12">
-              {this.createdLine()}
+              <div className="mb-1">{this.createdLine()}</div>
```

Comment:
`this.createdLine` already returns a div. Put the class name on that instead of making a new div.

Reply:
But I want it to only affect mobile. Any ideas how to do that?

Reply:
I believe you can use `mb-1 mb-md-0`. At least, I think margin classes are responsive.

Reply:
That totally did the trick. Thanks so much.

Reply:
Always look for Bootstrap classes before adding a element or a div